diff --git a/build.gradle.kts b/build.gradle.kts
index 50c70db..d975415 100755
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -44,6 +44,7 @@ repositories {
     maven("https://nexus.scarsz.me/content/groups/public")
     maven("https://libraries.minecraft.net")
     maven("https://repo.pgm.fyi/snapshots")
+    maven("https://jitpack.io")
 }
 
 dependencies {
@@ -55,9 +56,13 @@ dependencies {
     implementation("net.time4j:time4j-tzdata:5.0-2020a")
 
     compileOnly("app.ashcon:sportpaper:1.8.8-R0.1-SNAPSHOT")
-    implementation("app.ashcon.intake:intake-bukkit:1.2-SNAPSHOT")
+    // implementation("app.ashcon.intake:intake-bukkit:1.2-SNAPSHOT")
+    implementation("com.github.Electroid:intake:v1.2.1")
 
-    compileOnly("tc.oc.pgm:core:0.15-SNAPSHOT") {
+    /* compileOnly("tc.oc.pgm:core:0.15-SNAPSHOT") { */
+    /*     exclude(group = "fr.mrmicky", module = "FastBoard") */
+    /* } */
+    compileOnly("com.github.PGMdev:PGM:v0.15") {
         exclude(group = "fr.mrmicky", module = "FastBoard")
     }
     compileOnly("fr.mrmicky:fastboard:1.2.0")
diff --git a/src/main/kotlin/network/warzone/mars/report/ReportFeature.kt b/src/main/kotlin/network/warzone/mars/report/ReportFeature.kt
index 64e0e2f..dd713e5 100644
--- a/src/main/kotlin/network/warzone/mars/report/ReportFeature.kt
+++ b/src/main/kotlin/network/warzone/mars/report/ReportFeature.kt
@@ -9,7 +9,7 @@ import network.warzone.mars.utils.simple
 import org.bukkit.Bukkit
 import org.bukkit.event.EventHandler
 import org.bukkit.event.Listener
-import tc.oc.pgm.community.events.PlayerReportEvent
+// import tc.oc.pgm.community.events.PlayerReportEvent
 import java.util.*
 
 object ReportFeature : Feature<Report>(), Listener {
@@ -21,14 +21,14 @@ object ReportFeature : Feature<Report>(), Listener {
         TODO("Reports are not fetchable")
     }
 
-    @EventHandler
-    fun onReportCreate(event: PlayerReportEvent) = runBlocking {
-        val onlineStaff = Bukkit.getOnlinePlayers().filter { it.hasPermission("pgm.staff") }
-            .map { SimplePlayer(it.uniqueId, it.name) }.toSet()
+    /* @EventHandler */
+    /* fun onReportCreate(event: PlayerReportEvent) = runBlocking { */
+    /*     val onlineStaff = Bukkit.getOnlinePlayers().filter { it.hasPermission("pgm.staff") } */
+    /*         .map { SimplePlayer(it.uniqueId, it.name) }.toSet() */
 
-        ReportService.create(event.player.simple, event.sender.simple, event.reason, onlineStaff)
-    }
+    /*     ReportService.create(event.player.simple, event.sender.simple, event.reason, onlineStaff) */
+    /* } */
 }
 
 // Not a real construct
-data class Report(override val _id: UUID = UUID.randomUUID()) : Resource
\ No newline at end of file
+data class Report(override val _id: UUID = UUID.randomUUID()) : Resource
